### PR TITLE
add gatewatcher aioniq v2

### PR DESCRIPTION
**Pull request by CesarGW**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
This is the second release of the Gatewatcher AIONIQ (Partner contribution) content pack. This release add 8 new commands, and remove one. 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details: The command gw-list-alerts is removed.
   - [ ] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by ostolero** [comment]
Hi @CesarGW , 
thank you for contributing to our repo and for your patience!
I'm sorry this review has been delayed for so long I will try to reach it as soon as I can.

File: Packs/Gatewatcher-AionIQ/Integrations/GCenter/README.md [hunk 1]
```diff
@@ -1,284 +1,26 @@
 This integration allows, via about twenty commands, to interact with the GCenter appliance via its API.
 This integration was integrated and tested with version v2.5.3.102 of GCenter.
-To simplify GCenter v2.5.3.102 is called GCenter in the Pack.
+To simplify GCenter v2.5.3.102 is called GCenterv102 in the Pack.
```
Comment (ostolero) [inline Packs/Gatewatcher-AionIQ/Integrations/GCenter/README.md:3]: Was the name changes anywhere else? if there is a requirement to create a v2 integration? I don't think it's needed by the code changes.

**Review by ostolero — CHANGES_REQUESTED** [review]
Hi @CesarGW, fantastic work! covered all basis in the code and added unit tests and test playbook above and beyond 🥇  
Notice these few comments-
1. Our convention is to deprecate commands and not remove them. This is in order to support backwards compatibility. please return the deleted command.
2. I see the pr title is v2, is there are reason to create a v2 version to this integration?
Let me know if you have any questions :)
Thanks

File: Packs/Gatewatcher-AionIQ/Integrations/GCenter/README.md (hunk 1 above)
Comment (CesarGW) [reply Packs/Gatewatcher-AionIQ/Integrations/GCenter/README.md]: its simply a mistake because the integration was previously called GCenterv102

**Comment by CesarGW** [comment]
Hi @ostolero,

Thank you for your comments, we made the changes.
The title is called like that because there are many commands added, there is no particular reason.

File: Packs/Gatewatcher-AionIQ/Integrations/GCenter/GCenter.py [hunk 2]
```diff
@@ -275,49 +496,51 @@ def is_authenticated(self) -> bool:
             )
             return False
 
-    def list_alerts(self) -> dict:
```
Comment (ostolero) [inline Packs/Gatewatcher-AionIQ/Integrations/GCenter/GCenter.py:278]: revert the deletion of the `list_alerts` function

File: Packs/Gatewatcher-AionIQ/Integrations/GCenter/GCenter.py [hunk 3]
```diff
@@ -1274,14 +1907,14 @@ def main() -> None:
             return_results(
                 test_module(client=client)
             )
-        elif command == "gw-list-alerts":
-            return_results(
-                gw_list_alerts(client=client, args=args)
-            )
```
Comment (ostolero) [inline Packs/Gatewatcher-AionIQ/Integrations/GCenter/GCenter.py:1280]: revert

File: Packs/Gatewatcher-AionIQ/Integrations/GCenter/GCenter.py [hunk 4]
```diff
@@ -818,46 +1291,45 @@ def test_module(client: GwClient) -> str:  # noqa: E501
         return "Authentication error, please check ip/user/password/token: [ERROR]"
 
 
-def gw_list_alerts(client: GwClient, args: Dict[str, Any]) -> CommandResults:  # noqa: E501
```
Comment (ostolero) [inline Packs/Gatewatcher-AionIQ/Integrations/GCenter/GCenter.py:821]: revert the deletion of `gw_list_alerts` function

**Review by ostolero — CHANGES_REQUESTED** [review]
Hi @CesarGW , changes look good, just need to return the deleted code for the `gw-list-alerts` function. tried to do it myself but I don't have permission to your repo.
I'll reach out through slack so we can schedule a demo :)

**Review by ostolero — APPROVED** [review]
Looks great! let's schedule our demo 💯

**Comment by ostolero** [comment]
Demo was great, we can merge :)

**Pull request merged**